flow/payload.yaml

```diff
@@ -0,0 +1,48 @@
+components:
+  schemas:
+    Flow.Payload:
+      description: |-
+        A container for different types of payload, which is the data in the frame after protocol headers.
+        Some part of the payload will be overwritten with instrumentation data, contents and placement of 
+        which could be implementation specific.
+      type: object
+      properties:
+        choice:
+          description: |-
+            A choice used to determine the type of payload.
```

Comment:
"...the pattern of the bytes in the payload following the protocol headers . " maybe.

Reply:
I think we need to document the expected behaviour of each choice  . 
e.g. fixed would insert user defined pattern in the payload bytes. 
       increment_byte would set the first byte as 00 , second as 01 , upto FF and then the pattern would be repeated. 
       decrement_byte would set the first byte as FF , second as FE , 255th as 00 and then the pattern would be repeated. 
       increment_word would set the first 4 bytes as 00 00 00 00 , second as 00 00 00 01 and so on. 
      decrement_word would set the first byte as FF FF FF FF , second as FF FF FF FE and so on.